ui/search.html

```diff
@@ -23,79 +23,111 @@
         <div data-widget="topnavigation"><!-- --></div>
```

Comment:
I managed to get something to work by putting a couple styles in `oae.search.css` with an `!important` flag... :)

I'll take it out.